caktus_primer/static/js/modal.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+let clickEvents = ['click', 'mousedown', 'touchstart', 'dblclick']
+
+
+class Modal {
+    constructor(id) {
+        this.name = 'Modal'
+        this.options = {}
+        this.container = document.getElementById(id) || null
+        if (this.container) {
+            this.openTarget = this.container.querySelector('[aria-label="open"]')
+            this.dialogTarget = this.container.getElementsByTagName('dialog')[0]
+            this.closeTarget = this.container.querySelector('[aria-label="close"]')
+        }
+        else {
+            return
+        }
+
+        clickEvents.forEach((n) => (this.openTarget.addEventListener(n, (e) => { this.handleClick(e)})))
```

Comment:
Can we avoid nesting two arrow functions on the same line? Makes it hard to read.

Reply:
What specifically about this is hard to grok? Do you have any suggestions as to how to rewrite this? I will add a comment so that it is clearer.

Reply:
Do you have any suggestions as to how to rewrite this? I will add a comment so that it is clearer.

Reply:
not hard to grok, just hard to read.

myself, I would put each callback body on the next line. But its a pretty clear for event binding like it is, so i'm not going to insist.